src/style.css

```
@@ -1 +1,28 @@
 /* add styles here */
+body {
+margin: 0;
+}
+
+.stars {
+display: flex;
+justify-content: center;
+align-items: center;
+}
```

```suggestion
.stars {
  display: flex;
  justify-content: center;
  align-items: center;
}
```
looks like you forgot some spaces, let's fix it everywhere